django/forms/widgets.py

```diff
@@ -851,10 +851,16 @@ class SplitDateTimeWidget(MultiWidget):
     supports_microseconds = False
     template_name = 'django/forms/widgets/splitdatetime.html'
 
-    def __init__(self, attrs=None, date_format=None, time_format=None):
+    def __init__(self, attrs=None, date_format=None, time_format=None, date_attrs=None, time_attrs=None):
         widgets = (
-            DateInput(attrs=attrs, format=date_format),
-            TimeInput(attrs=attrs, format=time_format),
+            DateInput(
+                attrs=attrs if date_attrs is None else date_attrs,
```

Comment:
Sorry, I thought I had left a comment previously noting the need for a test for the `attrs if date_attrs is None else` logic (nothing fails if it's removed) but I guess I forgot to submit it.